calculation/sha_calc/sha_calc/directivity/bea20/directivity.py

```diff
@@ -0,0 +1,76 @@
+import math
+from pathlib import Path
+
+import numpy as np
+import pandas as pd
+
+from qcore import srf
+from IM_calculation.source_site_dist import src_site_dist
+from sha_calc.directivity.bea20 import bea20, utils
+
+
+def get_directivity_effects(
```

Comment:
I think `compute..` or `calculate...` might be clearer? `get..` sounds to me like the function is retrieving something